build.sbt

```diff
@@ -146,7 +146,7 @@ lazy val `play-json` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
       libraryDependencies ++= Seq(
         "org.scalatest"     %%% "scalatest"       % "3.2.18"   % Test,
         "org.scalatestplus" %%% "scalacheck-1-16" % "3.2.14.0" % Test,
-        "org.scalacheck"    %%% "scalacheck"      % "1.17.1"   % Test,
+        "org.scalacheck"    %%% "scalacheck"      % "1.17.0"   % Test,
```

Comment:
Need to downgrade because scalacheck patch release upgraded to sbt-scalajs 1.16 which is incompatible: https://github.com/typelevel/scalacheck/commit/8caf0b1de4bce75feefdc39c18886367c8530a03